src/styles/search-results.css

```diff
@@ -0,0 +1,31 @@
+p {
+  text-align: center;
+}
+
+.search-results {
+  margin-top: 30px;
+  display: flex;
+  flex-direction: row; 
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.search-results__images {
+  margin: 60px auto;
+  display: grid;
+  grid-template-columns: 3fr 3fr 3fr;
+  grid-gap: 10px;
+  justify-items: center;
+  align-items: center;
+  height: auto;
```

Comment:
just to let you know that height will be set to auto by default so you don't need to apply this CSS property